### PR TITLE
Pre-release cleaning

### PR DESCRIPTION
**Pull request by mereolog**
This pull requests defines the scope of 202301 release:

1. https://spec.industrialontologies.org/ontology/core/Core/
2. https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/
3. https://spec.industrialontologies.org/ontology/core/meta/MappingAnnotationVocabularyToCommons/
4. https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/

It also fixes a number of minor issues with the released ontologies and fills up the missing information.

### CODE REVIEW

File: AboutIOFDev.rdf [hunk 1]
```diff
@@ -28,7 +28,10 @@
 		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/20230109/AboutIOFDev/"/>
 		<iof-av:maturity rdf:resource="&iof-av;Provisional"/>
 		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/Core/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/maintenance/Maintenance/"/>
-		<owl:imports rdf:resource="https://www.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/"/>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"/>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/MappingAnnotationVocabularyToCommons/"/>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/"/>
+		<owl:imports rdf:resource="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/"/>
```
Comment (wsobel) [inline AboutIOFDev.rdf:34]: We deprecated the dev subdomain. We only use spec and use maturity status.

**Review by wsobel — CHANGES_REQUESTED** [review]
We should remove references to the dev subdomain: see https://oagiscore.atlassian.net/wiki/spaces/IOF/pages/4532142135/IRI+Structure+and+Format+V2.1#Development-IRI

**Review by wsobel — APPROVED** [review]
Changes look good.

**Review by ekwallace — APPROVED** [review]
Changes to maturity level and version IRIs look good.

-Evan

**Review by MilosDrobnjakovic — APPROVED** [review]
All looks good.

**Review by przemekgradzki — APPROVED** [review]
Looks good - see [Build #3](https://jenkins.industrialontologies.org/job/NIST/job/nist/view/change-requests/job/PR-26/3/)

**Pull request merged**